pretest.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var fs = require('fs');
+var path = require('path');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var sqlFile = path.resolve(__dirname, 'test', 'tables.dashdb.sql');
+
+if (!process.env.CI)
+  return console.log('Skipping pre-test ...');
+console.log('> Seeding the database ...');
+
+var contents = fs.readFileSync(sqlFile, {encoding: 'utf-8'});
+var lines = contents.trim().split(';');
```

Comment:
The `.execute()` function calls [ibm_db's query](https://github.com/strongloop/loopback-ibmdb/blob/master/lib/ibmdb.js#L254). I tried running the whole file as a whole but it only executes the first command. That is why I broke them out per line and executed each line separately.